index.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 var through = require('through2');
 
-module.exports = function () {
+module.exports = function (reversions) {
+	reversions = reversions || 1;
 	return through.obj(function (file, enc, cb) {
 		var history = file.history;
 
-		if (history.length > 1) {
+		if (history.length > reversions) {
```

Comment:
I don't think it should just silently not do its thing when there aren't enough history. Either throw or reverse as much as possible. Not sure which.